config.xml

```diff
@@ -1,19 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
`<tizen:category name="com.samsung.wmanager.WATCH_CLOCK"/>`

This line should still be in this file, as discussed with Mircea and Niels